frontend/src/components/NotifyModal.js

```diff
@@ -20,20 +10,27 @@ function NotifyModal() {
       return (
        <div className="notify-modal">
             <button className="notify-btn" onClick={() => setModalIsOpen(true)}>Notify Me!</button>
-            <Modal 
+            <ReactModal 
                 isOpen={modalIsOpen} 
-                onRequestClose={() => setModalIsOpen(false)} 
-                style={customStyles}
+                onRequestClose={() => setModalIsOpen(false)}
+                className="Modal"
+                overlayClassName="Overlay"
+                closeTimeoutMS={200} 
             >
-                <button className="close-btn" onClick={() => setModalIsOpen(false)}>X</button>
+                <div className="position-absolute top-0 end-0">
+                    <button className="close-btn" onClick={() => setModalIsOpen(false)}>
+                        <img className="xIcon" src={xIcon} alt="close button"></img>
+                    </button>
+                </div>
                 <div>
-                    <h3>Want to get updated?</h3>
-                    <p>Sign up to get alerts every time your favorite meal is featured at the dining hall!</p>
+                    <h5 className="modalTitle mb-0">Type in your email or phone number to access notification settings.</h5>
```

Comment:
this is the text to see the subscriptions page - can we have it say "Get notified for {{menu item}}" instead?

![image](https://user-images.githubusercontent.com/42781446/185067566-b053a4ca-cbd1-451b-ac6a-d9615d5841b6.png)